gen/calc.py

```diff
@@ -296,7 +366,18 @@ def calculate_exhibitor_static_ensemble(master_list):
         'ui_banner_header_content': 'null',
         'ui_banner_footer_content': 'null',
         'ui_banner_image_path': 'null',
-        'ui_banner_dismissible': 'null'
+        'ui_banner_dismissible': 'null',
+        'dcos_overlay_network': '{                    \
```

Comment:
It states we own / customers don't use several IP spaces.

It also says `'dcos_overlay_enable': 'true'` which both doesn't seem right (and breaks standard json practice...)